tests/panels/test_sql.py

```diff
@@ -104,6 +119,54 @@ async def test_recording_concurrent_async(self):
         # ensure the stacktrace is populated
         self.assertTrue(len(query["stacktrace"]) > 0)
 
+    @override_settings(DEBUG_TOOLBAR_CONFIG={"TRACK_DDT_MODELS": True})
+    def test_ddt_models_tracking(self):
+        self.assertEqual(len(self.panel._queries), 0)
+
+        sql_call_ddt()
+
+        # ensure query was logged
+        self.assertEqual(len(self.panel._queries), 1)
+        query = self.panel._queries[0]
+        self.assertEqual(query["alias"], "default")
+        self.assertTrue("sql" in query)
+        self.assertTrue("duration" in query)
+        self.assertTrue("stacktrace" in query)
```

Comment:
I think we may be better off replacing this with a check on the models' table in the sql query rather than these assertions.